UnraisinYourself/UI Extensions/View+AutoLayout.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  View+AutoLayout.swift
+//  UnraisinYourself
+//
+//  Created by Hannie Kim on 10/9/23.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+    func anchor(
+        top: NSLayoutYAxisAnchor? = nil,
+        leading: NSLayoutXAxisAnchor? = nil,
+        bottom: NSLayoutYAxisAnchor? = nil,
+        trailing: NSLayoutXAxisAnchor? = nil,
+        paddingTop: CGFloat = 0,
+        paddingLeading: CGFloat = 0,
+        paddingBottom: CGFloat = 0,
+        paddingTrailing: CGFloat = 0,
+        width: CGFloat? = nil,
+        height: CGFloat? = nil
+    ) {
+        translatesAutoresizingMaskIntoConstraints = false
+        
+        if let top {
+            topAnchor.constraint(equalTo: top, constant: paddingTop).isActive = true
+        }
+        
+        if let leading {
+            leadingAnchor.constraint(equalTo: leading, constant: paddingLeading).isActive = true
+        }
+        
+        if let bottom {
```

Comment:
Adds a negative value for you for bottom and trailing constraints when you want to add additional padding. That way, the callsite doesn't have to remember to add negative values for bottom and trailing